service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResource.java

```diff
@@ -92,7 +101,9 @@ public ApiResourceMetadata toApiMetadata() {
             .managedBy(managedBy.toApiModel())
             .privateResourceUser(
                 // TODO: PF-616 figure out how to supply the assigned user's role
-                new ApiPrivateResourceUser().userName(assignedUser));
+                new ApiPrivateResourceUser().userName(assignedUser))
```

Comment:
it's an optional field right now but I saw there's a todo to make it required. Do you want to handle null value here?

Reply:
I'm a little surprised we set a `ApiPrivateResourceUser` with a null `assignedUser` instead of just setting a null `ApiPrivateResourceUser`, but changing that would could break callers so I'd rather not do that in a large PR like this.